### PR TITLE
Update widget_modal.md

### PR DESCRIPTION
**Pull request by neeta-wagento**
## Purpose of this pull request

This pull request (PR) is for Type of Modal Widget is the Popup, not the slide

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_modal.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_modal.html
- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_modal.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/javascript-dev-guide/widgets/widget_modal.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ The element that triggers the modal.
 
 ### `type` {#modal_type}
 
-The type of window: popup or slide.
+The type of window: popup.
```
Comment (ronak2ram) [inline guides/v2.1/javascript-dev-guide/widgets/widget_modal.md]: Hi @neeta-wagento 
Thanks for your contribution.
For the modal type is popup or slider, both options are available to make a modal.
so If we remove `slide` then user not seen the second `slide` option.

**Comment by dobooth** [comment]
Can we handle this issue with this PR? https://github.com/magento/devdocs/issues/4524

**Comment by shrielenee** [comment]
@erikmarr We need to confirm what the applicable modal types are for this;  Is`slide` valid? Please see linked PR per Don above.

**Comment by erikmarr** [comment]
@neeta-wagento can you resolve the feedback from @ronak2ram? Also, please take a look at the following related issue as it pertains to your PR: https://github.com/magento/devdocs/issues/4524. Thank you!

**Comment by neeta-wagento** [comment]
Hello @erikmarr Somehow I could not add changes in this PR as its showing from the unknown repository at top, So I am closing this one and I have created new PR for https://github.com/magento/devdocs/pull/4538 the requested changes

**Pull request closed without merging**